lib/templates/_footer.html

```diff
@@ -4,9 +4,9 @@
     <div class="container-fluid">
         <div class="container">
             <p class="text-center">
-                <span class="no-break">{{^package.isSdk}}package{{/package.isSdk}} {{package.name}}</span>
-                &bull;
-                <span class="no-break">version {{package.version}}</span>
+                <img src="static-assets/favicon.png" alt="Dart" title="Dart"
+                  width="24" height="24" style="margin-right:8px">
+                <span>API docs for {{package.name}}, version {{package.version}}</span>
```

Comment:
I wanted , to read like "the package, at version". Using the bullet looked too ambiguous. It could have been read that the version of the dartdoc or the version of the output.